lib/streamlit/elements/markdown.py

```diff
@@ -117,7 +121,12 @@ def subheader(self, body):
 
         """
         subheader_proto = MarkdownProto()
-        subheader_proto.body = "### %s" % clean_text(body)
+        if anchor is None:
+            subheader_proto.body = '### %s' % clean_text(body)
+        else:
+            subheader_proto.body = '<h3 data-anchor="%s">%s</h3>' % (anchor, clean_text(body))
```

Comment:
Flyby: slight preference for the Python format string syntax
```
f"<h3 data-anchor="{anchor}">{clean_text(body)}</h3"
```
As it produces more readable code. I know we don't use it consistently in our codebase yet, but I'd like new code moving forward to use it, and to update existing references where appropriate 🙂